node-template/runtime/src/lib.rs

```diff
@@ -102,8 +96,9 @@ pub fn native_version() -> NativeVersion {
 
 parameter_types! {
 	pub const BlockHashCount: BlockNumber = 250;
-	pub const MaximumBlockWeight: Weight = 4 * 1024 * 1024;
-	pub const MaximumBlockLength: u32 = 4 * 1024 * 1024;
+	pub const MaximumBlockWeight: Weight = 1_000_000;
```

Comment:
As updated in the sheet, weight scale is now `[10, 1_000_00]` and the weight-to-fee factor `10^6`.

Reply:
1_000_000 seems small for a block. why not make it 1_000_000_000 per block to allow more fine-grained transaction weighting?

Reply:
Note that this also came hand in hand with the fact that a fixed multiplier of 1000 was removed from weights. So the limit hasn't actually changed at all.

Could easily be reverted, yet, my arguments being: 

- We already have a redundant factor of 10 (smallest weight is 10, changes are within a few hundred) as backup and _being more fine-grained_.
- A bit more readable to skim the weight annotations a and see `weight = 10, weight = 50, weight = 100` rather than a stack of zeros attached to all of them.

Reply:
@Kianenigma what about using powers of 2 instead?

Reply:
> We already have a redundant factor of 10 

Not enough.

> A bit more readable

Not a good enough reason, since this is actually important logic.

Reply:
> We already have a redundant factor of 10 

Not enough.

> A bit more readable

Not a good enough reason, since this is actually important logic.

Reply:
also, Po2 not a good idea here. weights are meant to be vaguely human-comprehensible, not things that are stored in memory or otherwise benefit from Po2 sizes.

Reply:
Reverted to a scale of `1_000_000_000`